pkgs/applications/editors/emacs/elisp-packages/libgenerated.nix

```diff
@@ -10,9 +13,13 @@ let
         ...
       }:
       { sha256, commit, ... }:
+      let
+        m = matchForgeRepo repo;
+      in
+      assert m != null;
       fetchFromGitHub {
-        owner = lib.head (lib.splitString "/" repo);
-        repo = lib.head (lib.tail (lib.splitString "/" repo));
+        owner = elemAt m 0;
```

Comment:
nit: maybe also add two functions `mkOwner = m: elemAt m 0` and `mkRepo`?

Reply:
That's more function calls for no added readability.